source/user-manual/reference/ossec-conf/localfile.rst

```diff
@@ -672,6 +678,46 @@ For example, to restrict syslog events related to a particular user name:
   The `eventchannel` format already provides a way to filter logs through queries. Therefore, `ignore` and `restrict` settings don't apply to this format.
 
 
+filter
+^^^^^^
+
+The `filter` tag is used to include PCRE2 regex filters for selectively collecting logs based on specific fields within `journald`.
+Each filter must specify a field and a regex pattern. The `ignore_if_missing` attribute can be used to indicate whether to ignore logs where the specified field is missing.
+
+
++--------------------+---------------------------------------------------------------+
+| **Default Value**  | n/a                                                           |
++--------------------+---------------------------------------------------------------+
+| **Allowed values** | Any `PCRE2 <regex.html#pcre2-syntax>`_ expression.            |
++--------------------+---------------------------------------------------------------+
+
+Use the `field` attribute to define in which journald field to apply the regex (Mandatory).
+
++-----------------------+--------------------------------------------------------------------------------------------------------------+
+| **ignore_if_missing** | When the attribute `ignore_if_missing` is set to `yes` it ignores the filter if the field does not exist.    |
++                       +------------------+-------------------------------------------------------------------------------------------+
+|                       | Default value    | no                                                                                        |
+|                       +------------------+-------------------------------------------------------------------------------------------+
+|                       | Allowed values   | no, yes                                                                                   |
++-----------------------+------------------+-------------------------------------------------------------------------------------------+
+
+
+Configuration example:
+
+.. code-block:: xml
+
+    <!-- For monitoring log files -->
+    <localfile>
+      <location>journald</location>
+      <log_format>journald</log_format>
+      <filter field="_SYSTEMD_UNIT">^cron.service$</filter>
+      <filter field="PRIORITY" ignore_if_missing="yes">[0-3]</filter>
+    <localfile>
+
+.. note::
+    Filters within the same `<localfile>` block follow an AND logic, while multiple blocks are evaluated in OR logic regarding log collection.
+
+
 Configuration examples
```

Comment:
I'm not sure about this, it would involve removing the syslog blocks with the configuration as these files are no longer collected as localfile but are collected by journald.